compiler/rustc_middle/src/dep_graph/dep_node.rs

```diff
@@ -1,16 +1,18 @@
-//! This module defines the `DepNode` type which the compiler uses to represent
-//! nodes in the dependency graph.
+//! Nodes in the dependency graph.
 //!
-//! A `DepNode` consists of a `DepKind` (which
+//! A node in the dependency graph is represented by a [`DepNode`].[^1]
+//! A `DepNode` consists of a [`DepKind`] (which
 //! specifies the kind of thing it represents, like a piece of HIR, MIR, etc)
```

Comment:
While you're at it, do you want to do this?

```suggestion
//! specifies the kind of thing it represents, like a piece of HIR, MIR, etc.)
```